tari-project/universe
# feat: keyring access & Monereo address generation

Description
---
Revamp the way we handle keyring access and use it to store the monero seed and the tari seed passphrase.

Motivation and Context
---
The user could deny access to the system Keychain, if they do we wouldn't be able to successfully store needed credentials for the app. Provide a wrapper to encapsulate storing the credentials in either the system keychain, or fallback to file IO storage. 

How Has This Been Tested?
---
Manually

What process can a PR reviewer use to test or verify this change?
---
Scenario 1:
- Install a slightly older version of TU. As of today (Friday 22nd, Nov)  this could be <=0.7.4.
- Assign a custom monero address / or leave the default monero address
- Then install the new version of the app
- Verify that your Monero address is your custom or the default monero address
- Verify no monero seed words are present

Scenario 2:
- Perform a completely fresh install of this version of TU
- Verify the monero address is already a custom address
- Verify you have access to the monero seed words





Breaking Changes
---
